client/trino-client/src/test/java/io/trino/client/TestIntervalDayTime.java

```diff
@@ -56,6 +56,8 @@ public void textMaxDays()
     public void testOverflow()
     {
         long days = (Long.MAX_VALUE / DAYS.toMillis(1)) + 1;
-        assertThatThrownBy(() -> toMillis(days, 0, 0, 0, 0)).isInstanceOf(IllegalArgumentException.class);
+        assertThatThrownBy(() -> toMillis(days, 0, 0, 0, 0))
```

Comment:
Please squash this commit with previous commit that introduced usage of `assertThatThrownBy`. Same for below.